components/badge/badge.css

```diff
@@ -0,0 +1,48 @@
+.badge{
+    position : relative;
+}
+
+.profile-badge{
+    bottom: 4px;
+    right: 1rem;
+    position: absolute;
+    height: 12px;
+    width: 12px;
+    border-radius: 50%;
+    border: 2px solid white;
+
+}
+
+.badge .online-badge{
+    background-color: var(--success-color);
+}
+
+.badge .offline-badge{
+    background-color: var(--grey);
+}
+
+.badge .dnd-badge{
+    background-color: var(--danger-color);
+}
+
+
```

Comment:
you could directly give styling as  .online-badge{
background-color: var(--grey);
}

Reply:
ok will do the changes mentioned!